### PR TITLE
Feature/add glog example

### PR DESCRIPTION
**Pull request by EstebanDugueperoux2**

### CODE REVIEW

File: tutorial/consuming_packages/glog_example/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from conans import ConanFile
+from conans import CMake
```
Comment (memsharded) [inline tutorial/consuming_packages/glog_example/conanfile.py:2]: Legacy, removed imports, don't use them

**Review by memsharded — COMMENTED** [review]
This is probably not the right repo for this. 
The idea of this repo is to be support for the docs, and be related to Conan features, not specific packages (that would need to be in the server).

Can you please clarify your intent with this PR?

**Comment by EstebanDugueperoux2** [comment]
> This is probably not the right repo for this. The idea of this repo is to be support for the docs, and be related to Conan features, not specific packages (that would need to be in the server).
> 
> Can you please clarify your intent with this PR?

It is about https://github.com/conan-io/conan-center-index/issues/11446, to get support.
Ok I'll keep my example in my fork.

**Comment by memsharded** [comment]
If it support related to Conan client behavior, you can send it to the main Conan repo. If it is something exclusively about the glog recipe writing, better in ``conan-center-index``.
You can report things to Conan repo, with a link to a reproducible case somewhere, but it is important to try to keep things as minimal as possible. If a thing can be reproduced with ``conan new hello/0.1 -m=cmake_lib`` package, much better, because otherwise, it could be a recipe issue and remain in conan-center-index. 

Thanks!

**Pull request closed without merging**